### PR TITLE
Add the mention of MY_PROJECTS() in TQL Cross-Tracker Search doc

### PR DESCRIPTION
**Pull request by robinsoc**
Part of [story #40088 Expert mode is the new default for cross tracker search](https://tuleap.net/plugins/tracker/?aid=40088)

### CODE REVIEW

**Review by Gashmob — APPROVED** [review]
Wait for approval of @Hyzual

File: languages/en/user-guide/tql/cross-tracker-search.rst [hunk 1]
```diff
@@ -257,6 +258,13 @@ Examples
     ORDER BY @last_update_date DESC
     // Get tickets assigned to me from support project. Display their title, status, who opened the ticket and the last modification date
 
+    SELECT @pretty_title, @submitted_by, @last_update_date, @status
+    FROM @project = MY_PROJECTS()
+    WHERE @status = OPEN() AND @assigned_to = MYSELF()
+    ORDER BY @last_update_date DESC
+    // Get artifacts opened and assigned to me from projects I'm member of,
```
Comment (Hyzual) [inline languages/en/user-guide/tql/cross-tracker-search.rst:265]:
```suggestion
    // Get artifacts with open status and assigned to me from projects I'm member of,
```

File: languages/en/user-guide/tql/cross-tracker-search.rst [hunk 2]
```diff
@@ -257,6 +258,13 @@ Examples
...
+    // display title, who opened the artifact, the last modification date and status
```
Comment (Hyzual) [inline languages/en/user-guide/tql/cross-tracker-search.rst:266]:
```suggestion
    // display title, who created the artifact, the last modification date and status
```

**Review by Hyzual — COMMENTED** [review]
Couple of nitpicks for the example, I'm fine with it if you don't think it's worth it

**Review by Hyzual — APPROVED** [review]

**Pull request merged**